packages/scm/src/browser/scm-tree-widget.tsx

```diff
@@ -718,7 +721,6 @@ export namespace ScmResourceFolderElement {
     export interface Props extends ScmElement.Props {
         treeNode: ScmFileChangeFolderNode;
         sourceUri: string;
-        path: string;
```

Comment:
Similarly to `ScmResourceGroupComponent.Props`, I believe we should document it as a breaking change.